pyamf/amf3.py

```diff
@@ -1093,13 +1097,14 @@ def readByteArray(self):
 
         buffer = self.stream.read(ref >> 1)
 
-        if buffer[0:2] == ByteArray._zlib_header:
```

Comment:
looks like I missed it from https://github.com/hydralabs/pyamf/commit/ce0440eba3220dc78d7643da56aedcc361e9f254 when merging the WK version